static/src/javascripts/projects/common/modules/ui/clickstream.js

```diff
@@ -6,11 +6,10 @@ import mediator from 'lib/mediator';
 type Options = {
     location?: Location,
     filter?: string[],
-    addListener?: boolean,
 };
 
-type Spec = {
-    el: ?Element,
+export type Spec = {
+    el?: Element,
```

Comment:
The `el` may be deleted, so this ought to be an optional field rather than a nullable field.